packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -251,19 +240,34 @@ class DDSTableOfContents extends StableSelectorMixin(LitElement) {
     }
   }
 
+  /**
+   * Handles intersection of target `<a>`s with the viewport by checking which target's
+   * immediate siblings are close to the viewport, and set the active target depending
+   * on their positions.
+   */
+  private _handleScroll = () => {
+    window.requestAnimationFrame(() => {
```

Comment:
```suggestion
    this.ownerDocument!.defaultView!.requestAnimationFrame(() => {
```

Suggest component DOM instead of global `window` whenever possible.